utils/report.h

```diff
@@ -54,6 +54,20 @@ struct uftrace_diff_policy {
 	bool full;
 };
 
+struct uftrace_report_call {
+	uint64_t addr;
+	struct list_head list;
+	struct list_head callers;
+	struct list_head callees;
+	int callers_size;
+	int callees_size;
+};
+
+struct uftrace_report_addr {
+	uint64_t addr;
+	struct list_head list;
+};
```

Comment:
If these two structs are only in implementation detail and not exposed to others, I'd move them to the .c file.